qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/car/MainCarSession.kt

```diff
@@ -62,18 +59,9 @@ class MainCarSession : Session() {
                     )
                 }
                 mapboxCarMap.setup(carContext, MapInitOptions(context = carContext))
-                mapboxCarMap.registerObserver(carMapStyleLoader)
-                val mapboxCarContext = MapboxCarContext(
-                    carContext = carContext,
-                    lifecycleOwner = this@MainCarSession,
-                    mapboxCarMap = mapboxCarMap,
-                ).also {
-                    this@MainCarSession.mapboxCarContext = it
-                }
-                mapboxCarContext.mapboxScreenManager.prepareScreens(mapboxCarContext)
-                mapboxNavigationManager = MapboxCarNavigationManager(carContext)
-                MapboxNavigationApp.registerObserver(mapboxNavigationManager)
-                MapboxNavigationApp.registerObserver(notificationInterceptor)
+                mapboxCarMap.registerObserver(carMapLoader)
+                MapboxNavigationApp.registerObserver(mapboxCarContext.mapboxNavigationManager)
+                MapboxNavigationApp.registerObserver(mapboxCarContext.notificationInterceptor)
```

Comment:
Looking to remove this. It's not the best way to enable the service, it would be better to enable it through an Option

Reply:
What I've tried to do is still a large change so I'm leaving it out of scope for this pull request